### PR TITLE
feat: Use new publication drawer in article details - EXO-72749_EXO-73246 - Meeds-io/MIPs#161

### PR DESCRIPTION
**Pull request by hakermi**
Use new publication drawer in article details

### CODE REVIEW

File: content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsEditPublishingDrawer.vue [hunk 1]
```diff
@@ -261,4 +261,4 @@ export default {
     },
   }
 };
-</script>
\ No newline at end of file
+</script>
```
Comment (azayati) [inline content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsEditPublishingDrawer.vue]: revert change

File: content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsDetails.vue [hunk 2]
```diff
@@ -73,9 +73,15 @@
         :cancel-label="$t('news.button.cancel')"
         @ok="deleteNews" />
       <exo-news-edit-publishing-drawer
-        v-if="news && currentUser"
+        v-if="news && currentUser && !newPublicationDrawerEnabled"
         :news="news"
         @refresh-news="getNewsById(newsId)" />
+      <note-publication-drawer
```
Comment (azayati) [inline content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsDetails.vue:79]: this component should be loaded only when FF is active

**Review by azayati — CHANGES_REQUESTED** [review]

File: content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsDetails.vue (hunk 2 above)
Comment (hakermi) [reply content-webapp/src/main/webapp/vue-app/news-details/components/ExoNewsDetails.vue:79]: Done

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**